_common/KeymanHosts.php

```diff
@@ -119,14 +119,16 @@ function __construct() {
         // TODO: allow override of these with e.g. KEYMANHOSTS_API_KEYMAN_COM='https://api.keyman.com';
         $this->s_keyman_com = "{$site_protocol}s.keyman.com{$site_suffix}";
         $this->api_keyman_com = "{$site_protocol}api.keyman.com{$site_suffix}";
-        // $this->api_keyman_com = "{$site_protocol}api.keyman.com"; //{$site_suffix}";
         $this->help_keyman_com = "{$site_protocol}help.keyman.com{$site_suffix}";
         $this->downloads_keyman_com = "{$site_protocol}downloads.keyman.com{$site_suffix}";
         // $this->downloads_keyman_com = "{$site_protocol}downloads.keyman.com"; //{$site_suffix}";
         $this->keyman_com = "{$site_protocol}keyman.com{$site_suffix}";
         $this->keymanweb_com = "{$site_protocol}keymanweb.com{$site_suffix}";
         $this->r_keymanweb_com = "https://r.keymanweb.com"; /// local dev domain is usually not available
       }
+      // temp overrides
+      $this->api_keyman_com = "http://localhost:8098";
+      $this->downloads_keyman_com = "https://downloads.keyman.com";
```

Comment:
@mcdurdin - open to suggestions on how to handle the local "Docker" sites no longer named `.local`.

Reply:
I think we setup two things:

1. A docker image that reverse proxies to the various keyman.com domains from a single port.
2. Have an env var that specifies the port and protocol we want to use for that reverse proxy, defaulting to http + 8090; we set the env var to https + 443 on the live+staging environments.

Then each site can have its defined port, the reverse proxy has a record of which port is where, and we can use keyman-local.com as our local version:

* http://keyman-local.com:8090/
* http://s.keyman-local.com:8090/
* http://downloads.keyman-local.com:8090/
* http://web.keyman-local.com:8090/
* http://help.keyman-local.com:8090/
* http://api.keyman-local.com:8090/

On my machine, I may even decide to setup the reverse proxy to listen on :80, so I can skip the port altogether when using the local site.

We already own keyman-local.com.

Reply:
I think these are now working with darcywong00/reverse-proxy.

I left some tweaks on KeymanHosts.php
local site protocol 'http://'
downloads uses the live https://downloads.keyman.com

Reply:
I wonder if we can move the reverse-proxy repo into keymanapp. Call it website-local-proxy perhaps? We'll all need it...

Reply:
maybe it can go in keymanapp/resource/devbox/website-local-proxy/ ?

Reply:
no, I think a separate repo is fine, but in the keymanapp org.